paymentsheet/src/test/java/com/stripe/android/lpmfoundations/luxe/LpmRepositoryTestHelpers.kt

```diff
@@ -0,0 +1,44 @@
+package com.stripe.android.lpmfoundations.luxe
+
+import com.stripe.android.lpmfoundations.paymentmethod.PaymentMethodMetadataFactory
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.CardDefinition
+import com.stripe.android.model.StripeIntent
+import com.stripe.android.ui.core.BillingDetailsCollectionConfiguration
+import com.stripe.android.ui.core.R
+import com.stripe.android.ui.core.elements.LayoutSpec
+
+object LpmRepositoryTestHelpers {
+    val card: SupportedPaymentMethod = CardDefinition.hardcodedCardSpec(BillingDetailsCollectionConfiguration())
+
+    val usBankAccount: SupportedPaymentMethod = SupportedPaymentMethod(
+        code = "us_bank_account",
+        requiresMandate = true,
+        displayNameResource = R.string.stripe_paymentsheet_payment_method_us_bank_account,
+        iconResource = R.drawable.stripe_ic_paymentsheet_pm_bank,
+        lightThemeIconUrl = null,
+        darkThemeIconUrl = null,
+        tintIconOnSelection = true,
+        requirement = USBankAccountRequirement,
+        formSpec = LayoutSpec(emptyList())
+    )
+}
+
+fun LpmRepository.updateFromDisk(stripeIntent: StripeIntent) {
```

Comment:
The fact that we need these is a huge testament to how much we rely on this and the mutation/side effects it creates.

This is more or less trying to reduce the test changes for this PR, but long term we should make the data flow through, rather than rely on mutation. Hoping to tackle this in a follow up.